test/utils/node-http.js

```diff
@@ -3,7 +3,10 @@
 const util = require('node:util')
 
 function closeServerAsPromise (server) {
-  return () => util.promisify(server.close.bind(server))()
+  return () => {
+    server.closeIdleConnections()
```

Comment:
Is this needed?

Reply:
I tried to change the order of the `t.after` in a test but there a couple others exhibiting the same behavior. I therefore chose to act at the source rather than each specific case.